### ISSUE TITLE
Track item effect

### ISSUE THREAD

**Issue by Himuraken** [ennvina/spellactivationoverlay#94]
Hello. Is there a way to track an item effect like the Soul Preserver trinket ?

![image](https://user-images.githubusercontent.com/89361868/197076646-f134749e-dd19-4797-8c74-a0ab996b981d.png)

i want this to trigger the same effect as Omen of clarity.

I tried a few Weak aura but it dont feel the same as the addon.

![image](https://user-images.githubusercontent.com/89361868/197076527-31e85912-6da6-429f-a36a-4fdb4667dc4c.png)

![image](https://user-images.githubusercontent.com/89361868/197076567-775c0d99-a759-4b98-808a-1c9e0588f3aa.png)

**Comment by ennvina** [comment]
You can add this effect, granted you are brave enough to edit the source code.

I will give you the steps to do so, but you have to know 3 things;
- the effect may overlap with existing effects, so it may get confusing depending on other effects that already exist
- I haven't tested the code because I don't have this trinket myself
- this code shall be reset when you download an update of the addon, in which case you will need to re-do these steps

The goal is pretty simple: edit the addon source code (with a text editor like VS Code, or even Notepad), save the modified file, and reload the addon (e.g., reload UI, log out / log in, etc.)

You will need to edit the source code of the class you're playing with. If you have the trinket on multiple characters, simply edit as many files as needed.

The source code is in the addon folder, in `WoW/Interface/AddOns/SpellActivationOverlay`

All modifications happen in the `registerClass` function of each class.

**Druid**

You must edit `classes\druid.lua` and add this line inside `registerClass`:

```lua
self:RegisterAura("healing_trance", 0, 60512, "natures_grace", "Left + Right (Flipped)", 1, 255, 255, 255, true);
```

**Paladin**

You must edit `classes\paladin.lua` and add this line inside `registerClass`:

```lua
self:RegisterAura("healing_trance", 0, 60513, "natures_grace", "Left + Right (Flipped)", 1, 255, 255, 255, true);
```

**Priest**

You must edit `classes\priest.lua` and add this line inside `registerClass`:

```lua
self:RegisterAura("healing_trance", 0, 60514, "natures_grace", "Left + Right (Flipped)", 1, 255, 255, 255, true);
```

**Shaman**

You must edit `classes\shaman.lua` and add this line inside `registerClass`:

```lua
self:RegisterAura("healing_trance", 0, 60515, "natures_grace", "Left + Right (Flipped)", 1, 255, 255, 255, true);
```

PS. even though all the above lines look the same, there is one thing that differs for each class: the spell ID (druid is 60512, paladin is 60513, etc.)

**Comment by ennvina** [comment]
Coming a bit late, but this has been implemented as a separate spell alert, similar to Mage's Clearcasting effect.

**Comment by ennvina** [comment]
Hi, closing this issue as it was implemented in #149 and released in [v0.9.5](https://github.com/ennvina/spellactivationoverlay/releases/tag/v0.9.5). Thanks for the feedback!
